augur/curate/format_dates.py

```diff
@@ -36,11 +36,13 @@ def register_parser(parent_subparsers):
 
     optional = parser.add_argument_group(title="OPTIONAL")
     optional.add_argument("--expected-date-formats", nargs="+", action="extend",
-        default=DEFAULT_EXPECTED_DATE_FORMATS,
-        help="Expected date formats that are currently in the provided date fields, " +
-             "defined by standard format codes as listed at " +
-             "https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes. " +
-             "If a date string matches multiple formats, it will be parsed as the first matched format in the provided order.")
+        help=f"""Custom date formats for values in the provided date fields, defined by standard
```

Comment:
_super minor nit_

Maybe this has been discussed before, but I just noticed that multiline help strings are formatted oddly in the docs

![Screenshot 2025-01-13 at 7 52 00 PM](https://github.com/user-attachments/assets/3e7bcd90-1663-45a5-8d9a-0fb180bd4d6c)

Ensuring that the first line has similar indentation as the following lines (e.g. filter's [--query-columns help](https://github.com/nextstrain/augur/blob/1843a9b02d5380e81d4adb874db85704985670b4/augur/filter/__init__.py#L34)) seems to remove the bolding, although they still have an extra indent in the docs.

Reply:
Good catch! I've addressed this with 49568ad0aab46d80e305e763ff7cd225af91b4f4. Created an issue for further discussion: #1720